History.markdown

```diff
@@ -1,3 +1,9 @@
+### HEAD
+
+### Minor Enhancements
+
+* [3.x] kramdown v2 upgrade (#8322)
```

Comment:
is the `[3.x]` distinction needed here?
would be self-explanatory when the heading gets changed to `## v3.9.0 ...` on release

Reply:
It's just what's automatic – takes the title of the PR and its number and adds it in there. Sure, we could take it out if you want.